thunder/tests/framework.py

```diff
@@ -232,10 +233,20 @@ def version(self):
         return torch.__version__
 
 
+class DynamoThunderTestExecutor(TestExecutor):
```

Comment:
This adds a test executor for Dynamo (which is cool), but doesn't register it, which means that tests generated to use test executors won't instantiate a variant for it.

That seems totally reasonable for now so we don't increase test time. Would you add a comment to this describing that it isn't returned from `_all_test_executors()` intentionally.

What are your thoughts on testing this variant? Did you have a chance to run the tests locally?

Reply:
Thanks, I added a comment that this TestExecutor is not included in the `_all_test_executors` list and that test instantiation needs to opt-in to enable testing with this executor.

This variant is tested for some end-to-end tests we have in test_networks.py also it's indirectly tested through running nightly benchmarks (not part of the GitHub CI).